src/main/java/net/snowflake/client/core/SystemUtil.java

```diff
@@ -0,0 +1,33 @@
+package net.snowflake.client.core;
+
+import static net.snowflake.client.jdbc.SnowflakeUtil.systemGetProperty;
+
+import net.snowflake.client.log.SFLogger;
+import net.snowflake.client.log.SFLoggerFactory;
+
+public class SystemUtil {

```

Comment:
this class does not have to be public - let's remove the modifier

Reply:
Removed